### PR TITLE
added staking

### PR DESCRIPTION
**Pull request by martinfridrich**
#84

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,13 +41,14 @@ cargo build --release
 Purge any existing dev chain state:
 
 ```bash
-./target/release/hack-hydra-dx purge-chain --dev
```
Comment (jak-pan) [inline README.md:44]: It was possible to run solo chain with babe, is this not possible anymore?

Comment (martinfridrich) [reply README.md]: yes, it works. I've fixed readme in next commit

File: runtime/src/constants.rs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	/// <https://research.web3.foundation/en/latest/polkadot/BABE/Babe/#6-practical-results>
+
+	pub const MILLISECS_PER_BLOCK: Moment = 10000;
+	pub const SECS_PER_BLOCK: Moment = MILLISECS_PER_BLOCK / 1000;
+
+	pub const MINUTES: BlockNumber = 60 / (SECS_PER_BLOCK as BlockNumber);
+	pub const HOURS: BlockNumber = MINUTES * 60;
+	pub const DAYS: BlockNumber = HOURS * 24;
+
+	pub const SLOT_DURATION: Moment = MILLISECS_PER_BLOCK;
```
Comment (enthusiastmartin) [inline runtime/src/constants.rs:41]: In the lib.rs - there are similar constants ( mils per block, duration, minutes etc )

Might be better to consolidate and have it only in one place.

File: node/Cargo.toml [hunk 3]
```diff
@@ -30,30 +30,40 @@ module-amm-rpc = { path = '../pallets/amm/rpc', version = '2.0.0' }
...
-sc-keystore = '2.0.0'
+sc-service = '0.8.0'
 sc-transaction-pool = '2.0.0'
 sp-api = '2.0.0'
+sp-authority-discovery = '2.0.0'
 sp-block-builder = '2.0.0'
 sp-blockchain = '2.0.0'
 sp-consensus = '0.8.0'
-sp-consensus-aura = '0.8.0'
+sp-consensus-babe = '=0.8.0'
```
Comment (enthusiastmartin) [inline node/Cargo.toml:60]: Probably the equal sign should be removed ?

**Comment by jak-pan** [comment]
LGTM

**Review by enthusiastmartin — APPROVED** [review]

**Pull request merged**